src/components/util/ask-text-dialog.js

```diff
@@ -62,16 +62,16 @@ export const AskTextDialog = ({ title, value, show, onValidate, onClose }) => {
                 )}
             </DialogContent>
             <DialogActions>
-                <Button onClick={handleClose}>
-                    {intl.formatMessage({ id: 'cancel' })}
-                </Button>
                 <Button
                     variant={'outlined'}
                     onClick={handleValidate}
                     disabled={!isNameOK}
                 >
                     {intl.formatMessage({ id: 'validate' })}
                 </Button>
+                <Button onClick={handleClose}>
```

Comment:
We must have the same order everywhere, maybe it was already changed. keep same order, cancel then validate at far right

Reply:
done